saleor/graphql/order/mutations/draft_order_update.py

```diff
@@ -219,7 +225,11 @@ def _save(
             address_fields = save_addresses(instance, cleaned_input)
             updated_fields.extend(address_fields)
 
-            if "shipping_method" in cleaned_input:
+            is_shipping_method_update_required = (
+                "shipping_method" in cleaned_input
+                and cleaned_input["shipping_method"] != instance.shipping_method
+            )
+            if is_shipping_method_update_required:
```

Comment:
```suggestion
            shipping_method_update_required = (
                "shipping_method" in cleaned_input
                and cleaned_input["shipping_method"] != instance.shipping_method
            )
            if shipping_method_update_required:
```

Reply:
Hmm, what is wrong with adding `is_` prefix? For me this is clear indication that the variable is boolean type.

Reply:
It has a suffix `_update_required` which implied boolean to me (0 / 1 something either requires an update or not, it can't be "a little bit required" 😄 ) but I see that in Saleor we commonly use `is_` prefix, let's leave it as is.